### ISSUE TITLE
Allow opening *.Diagsession files on machines without Visual Studio. 

### ISSUE THREAD

**Issue by vancem** [Microsoft/perfview#277]
If you try to open a *.diagsession file on a machine without Visual Studio installed you will get the following error.  
```
Exception Occurred: System.ArgumentNullException: Value cannot be null.
Parameter name: pathToNativePackagingDll
   at Microsoft.DiagnosticsHub.Packaging.InteropEx.DhPackage.GetPackageFactory(String pathToNativePackagingDll)
   at Microsoft.DiagnosticsHub.Packaging.InteropEx.DhPackage.LoadAndOpen(String pathToPackageAssembly, String pathToPackage)
   at Microsoft.DiagnosticsHub.Packaging.InteropEx.DhPackage.Open(String pathToPackage)
```

I turns out that we don't have a complete set of Diagsession support Dlls as part of PerfVIew.   It happens to work if you have VS installed, because it has logic to go see if VS is installed and use that if possible.  

The missing file is 

```
C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\CommonExtensions\Platform\DiagnosticsHub\x86\DiagnosticsHub.Packaging.dll"
```
You want a amd64 version as well.  

If we  add these to the distribution it should fix this.

**Comment by vancem** [comment]
We should upgrade the Diagsession DLLs while we are at it.  This requires updating the PerfView support packages, which is why I assigned this to myself.

**Comment by vancem** [comment]
Fixed by PR #278
